### PR TITLE
Add Japanese language to w3c_website_templates_bundle+intl-icu.ja.yaml

### PR DESCRIPTION
**Pull request by takuan517**
I have added Japanese to w3c_website_templates_bundle+intl-icu.ja.yaml.

### CODE REVIEW

**Comment by takuan517** [comment]
@jaykishigami
I submitted a Pull Request for your translation.
I think I found a few corrections, so I will comment on them.

File: translations/w3c_website_templates_bundle+intl-icu.ja.yaml [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-            ja: __page.lang.targetname.ja
-            zh-hans: __page.lang.targetname.zh-hans
-    default_title: __page.default_title
+            en: 英語のページ
+            ja: 日本語のページ
+            zh-hans: 中国語のページ
+    default_title: World Wide Web Consortium (W3C)
     default_description: >-
-        __page.default_description
+        ワールド・ワイド・ウェブ・コンソーシアム(W3C)は、メンバー組織、専任スタッフ、そして利用者が協力してウェブ標準を開発する国際的なコミュニティです。
```
Comment (takuan517) [inline translations/w3c_website_templates_bundle+intl-icu.ja.yaml:11]: ワールド・ワイド・ウェブ・コンソーシアム → World Wide Web Consortiumの方が良いかなと思います．

File: translations/w3c_website_templates_bundle+intl-icu.ja.yaml [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
     skip_to_content: コンテンツにスキップ
     translations: >-
-        他の言語:
-    translationsarialabel: __page.translations.arialabel
-    toc: __page.toc
-    related_content: __page.related
+       <span lange="jp">このページを読んでください:</span>
```
Comment (takuan517) [inline translations/w3c_website_templates_bundle+intl-icu.ja.yaml:14]: 中国語の方は，`切换语言` となっているので，このページをどの言語で読むかというニュアンスなのかなと思っています．（適切な表現がわかっていませんが，，，，)

File: translations/w3c_website_templates_bundle+intl-icu.ja.yaml [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
+    translationsarialabel: 他の言語
+    toc: ページの内容
+    related_content: このセクションでは
```
Comment (takuan517) [inline translations/w3c_website_templates_bundle+intl-icu.ja.yaml:17]: 中国語の方は，相关内容(関連コンテンツという意味らしいです)となっているので，別に適切な表現があるかもですね，，

**Review by takuan517 — COMMENTED** [review]
いくつかコメントをしました．

**Comment by japuiglaucia** [comment]
Add Japanese language to w3c_website_templates_bundle+intl-icu.ja.yaml #129

**Comment by vivienlacourba** [comment]
@takuan517 @jaykishigami @yonaomi any progress on addressing Takumi's comments which If I see correctly are addressed to Jay specifically?
Once they will be addressed and you believe this Pull Request is ready, then please remove the "Draft" status and assign it to @jean-gui who will then be able to move forward.

**Comment by takuan517** [comment]
@vivienlacourba 
Thank you for your comment.
We are doing this translation only by looking at the source code. Therefore, there is a high possibility that we are providing incorrect Japanese translations.
A few days ago, I sent an email as well, but I would like to obtain permission for the CMS to run this website in a local environment for verification purposes.

File: translations/w3c_website_templates_bundle+intl-icu.ja.yaml [hunk 4]
```diff
@@ -44,102 +44,106 @@ footer:
...
+    next: 次のページ
+    last: 前回のページ
+    page: "ページ {page}"
     current_page: "{page}"
-    summary: __pagination.summary {page} {from}-{to}
-ecosystems: __ecosystems {count}
+    summary:  >-
+        {to, select,
+            0 {There are no results}
+            other {このページは {page}, 提示されているページは {from}-{to}}
```
Comment (himorin) [inline translations/w3c_website_templates_bundle+intl-icu.ja.yaml:74]: 表示？

File: translations/w3c_website_templates_bundle+intl-icu.ja.yaml [hunk 5]
```diff
@@ -44,102 +44,106 @@ footer:
...
     blog:
         filters:
-            search: __listing.blog.filters.search
-            search_button: __listing.blog.filters.search_button
-            all: __listing.blog.filters.all
-            categories: __listing.blog.filters.categories
-            categories_label: __listing.blog.filters.categories_label
-            archives: __listing.blog.filters.archives
-            archives_label: __listing.blog.filters.archives_label
+            search: _投稿の検索
```
Comment (himorin) [inline translations/w3c_website_templates_bundle+intl-icu.ja.yaml:80]:
```suggestion
            search: 投稿の検索
```

**Comment by takuan517** [comment]
After resolving the conflicts, the file differences became larger, so I will create a new Pull Request.

**Comment by takuan517** [comment]
Here is the link to the new Pull Request: https://github.com/w3c/w3c-website-templates-bundle/pull/133
I will go ahead and close this Pull Request.

**Pull request closed without merging**